release-team/role-handbooks/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
+
+- Make sure that docs PRs for the release have the correct base and set the correct Milestone. For example, feature PRs for version 1.11 need a base branch of `release-1.11` and the Milestone set to `1.11`.
+
+### Maintain the current and upcoming release branches
+
+- Periodically merge `master` into the current release branch.
+
+For example, during the v1.11 release cycle, you must periodically merge `master` into `release-1.10`.
+
+Periodic merges must continue throughout the release cycle, until the pending release (for example, `release-1.11`) merges to `master`. This is to ensure that the current release branch remains up to date, and to preserve the current branch as a snapshot when the new version is released. Periodic maintenance also helps avoid excessive merge conflicts between `master` and the current branch at the end of the release cycle.
+
+On your local fork:
+```
+$ git fetch upstream
+$ git checkout release-<current>
+$ git merge upstream/master
+$ git push -f upstream release-<current>
+```
+
+- Periodically rebase the upcoming release branch onto `master`.
+
+For example, during the v1.11 release, you must periodically rebase `release-1.11` onto `master` in order to pick up any changes in `master` during the release process.
+
+On your local fork:
+```
+$ git fetch upstream
+$ checkout release-<upcoming>
+$ git rebase upstream/master
+$ git push -f upstream release-<upcoming>
+```
+
+### Stay connected with the release cycle
+
+- Attend #sig-release meetings.
+
+    Meetings increase in frequency over the course of the release, moving from weekly to daily.
+
+### Work with feature developers and reviewers
+
+- Keep developers apprised of due dates for docs.
+
+    Send announcements to:
+        - [gke-kubernetes-team@google.com](mailto:gke-kubernetes-team@google.com)
+        - [kubernetes-sig-release@google.com](mailto:kubernetes-sig-release@google.com)
+        - [#sig-release](https://kubernetes.slack.com/messages/C2C40FMNF/) on Slack
+        - [kubernetes-sig-leads@google.com](mailto:kubernetes-sig-leads@google.com)
+        - Feature owners listed in the OSS feature tracking spreadsheet
+
+- As developers open PRs, give them `sig/*` labels (listed in the feature tracking spreadsheet) to help find technical reviewers.
+
+### Review PRs
+
+- Remind tech reviewers to comment with `/lgtm` when PRs pass review.
+
+- Ask SIG Docs (#sig-docs channel on Slack) for help with reviews.
+
+- Review PR commit histories.
+
+    Make sure that PR commit histories contain only changes to files. It's especially important to avoid revert commits.
+
+    Encourage developers to rebase any PRs with complicated commit histories.
 
 ### Last minute PRs
 
-*   Triage: 
+*   Triage:
     *   Decide what must go out with release, what can be published just after the release.
```

Comment:
Do we have a exception process in place for features with late incoming doc PRs? If not we can work on something for 1.13. 

Basically I would like to somehow emphasize in the verbiage here that we will and should punt features that are delayed on critical doc PRs that need to go with the release.  We should treat the final doc PR deadline as a hard one one